app/src/main/java/com/iron/espresso/presentation/home/mystudy/StudyDetailActivity.kt

```diff
@@ -0,0 +1,83 @@
+package com.iron.espresso.presentation.home.mystudy
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.google.android.material.tabs.TabLayoutMediator
+import com.iron.espresso.R
+import com.iron.espresso.ToolbarHelper
+import com.iron.espresso.databinding.ActivityStudyDetailBinding
+import com.iron.espresso.presentation.home.mystudy.studydetail.ChattingFragment
+import com.iron.espresso.presentation.home.mystudy.studydetail.NoticeFragment
+import com.iron.espresso.presentation.home.mystudy.studydetail.StudyInfoFragment
+
+class StudyDetailActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityStudyDetailBinding
+    private lateinit var toolbarHelper: ToolbarHelper
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_study_detail)
+
+        toolbarHelper = ToolbarHelper(this, binding.appbar).apply {
+            setTitle(intent.extras?.getString("title"))
```

Comment:
```suggestion
            setTitle(intent.extras?.getString(EXTRA_TITLE))
```
"title" 이거 companion object 에 상수로 선언해서 관리하면 좋을 것 같아요

Reply:
수정했습니다